lib/ansible/modules/windows/win_copy.py

```diff
@@ -44,59 +33,92 @@
     - Remote absolute path where the file should be copied to. If src is a
       directory, this must be a directory too.
     - Use \ for path separators or \\ when in "double quotes".
+    - If C(dest) ends with \ then source or the contents of source will be
+      copied to the directory without renaming.
+    - If C(dest) is a nonexistent path, it will only be created if C(dest) ends
+      with "/" or "\", or C(src) is a directory.
+    - If C(src) and C(dest) are files and if the parent directory of C(dest)
+      doesn't exist, then the task will fail.
     required: true
   force:
     version_added: "2.3"
     description:
-    - If set to C(yes), the remote file will be replaced when content is
-      different than the source.
-    - If set to C(no), the remote file will only be transferred if the
+    - If set to C(yes), the file will only be transferred if the content
+      is different than destination.
+    - If set to C(no), the file will only be transferred if the
       destination does not exist.
-    default: True
-    choices:
-    - yes
-    - no
+    default: 'yes'
+    type: bool
+  local_follow:
+    version_added: '2.4'
+    description:
+    - This flag indicates that filesystem links in the source tree, if they
+      exist, should be followed.
+    default: 'yes'
+    type: bool
   remote_src:
     description:
     - If False, it will search for src at originating/master machine, if True
       it will go to the remote/target machine for the src.
-    default: False
-    choices:
-    - True
-    - False
+    default: 'no'
+    type: bool
     version_added: "2.3"
   src:
     description:
     - Local path to a file to copy to the remote server; can be absolute or
-      relative. If path is a directory, it is copied recursively. In this case,
-      if path ends with "/", only inside contents of that directory are copied
-      to destination. Otherwise, if it does not end with "/", the directory
-      itself with all contents is copied. This behavior is similar to Rsync.
+      relative.
+    - If path is a directory, it is copied (including the source folder name)
+      recursively to C(dest).
+    - If path is a directory and ends with "/", only the inside contents of
+      that directory are copied to the destination. Otherwise, if it does not
+      end with "/", the directory itself with all contents is copied.
+    - If path is a file and dest ends with "\", the file is copied to the
+      folder with the same filename.
     required: true
 notes:
 - For non-Windows targets, use the M(copy) module instead.
-author: "Jon Hawkesworth (@jhawkesworth)"
+- Currently win_copy does not support copying symbolic links from both local to
+  remote and remote to remote.
+- It is recommended that backslashes C(\) are used instead of C(/) when dealing
+  with remote paths.
+author:
```

Comment:
Should we add here that the win_copy module is not a very efficient transfer mechanism and if possible using HTTP downloading using get_url or uri is preferred.